HelpMyStreetFE/HelpMyStreetFE/Views/RequestHelp/EditorTemplates/RequestHelpDetailStageViewModel.cshtml

```diff
@@ -2,6 +2,7 @@
 @using HelpMyStreet.Utils.Enums;
 
 <input type="hidden" name="DetailStep" class="model" value='@Json.Serialize(Model)' />
+<input type="hidden" name="NeedBothNames" class="model" value="@Model.NeedBothNames.ToString()" />
```

Comment:
Not such a fan of using an input to pass this value to the front end.  It would be better as an attribute on the name fields.  This would also pave the way for having different validation for Requestor/Recipient, if required on FE-774.